firmware/brake/kia_soul_ps/brake_control_module.ino

```diff
@@ -698,28 +693,32 @@ void brakeUpdate()
             }
 
 
-            // if driver is not braking, transition to wait state
-            if( pressure_req <= ZERO_PRESSURE)
-            {
-                DEBUG_PRINT("pressure request below threshold");
-                brakeStateMachine.transitionTo( Wait );
-            }
-        }
-    }
-
 
-    //
-    void brakeExit()
+        }
+    } else if (pressure_req <= ZERO_PRESSURE) 
```

Comment:
The `else if ( ... )` needs to be on a newline, to match the style guide. Also spaces are required between the `(` and `)` in the `if`statements. This is in Section 12 of `CONTRIBUTING.md`.